packages/components/src/docs/Migration/ui-qrcode.mdx

```diff
@@ -0,0 +1,5 @@
+# Migration Guide: Deprication of `ui-qrcode`
+
+## [`ui-qrcode`] is deprecated
+
+As the ui-qrcode component has seen little use, it will be deprecated and removed from the SDS Library. Should you require a QR code component, we recommend using the [Shoelace QR Code component](https://shoelace.style/components/qr-code) as a replacement.
```

Comment:
```suggestion
As the ui-qrcode component has seen little use, it will be deprecated and removed from the SDS Library. Should you require a QR code component, please open up a feature request and we recommend using the [Shoelace QR Code component](https://shoelace.style/components/qr-code) meanwhile as a replacement.
```

Reply:
I would like to verify this with @mariohamann as we don't need the suggestions above, if there are technical reasons and also should add them here.

Reply:
The important thign is really more "easy to implement with third-party library or other web component libraries, no benefit from Design side and we want to prevent bigger bundle sizes".